gensim/models/ldamodel.py

```diff
@@ -567,15 +567,18 @@ def init_dir_prior(self, prior, name):
         if isinstance(prior, six.string_types):
             if prior == 'symmetric':
                 logger.info("using symmetric %s at %s", name, 1.0 / self.num_topics)
-                init_prior = np.asarray([1.0 / self.num_topics for i in xrange(prior_shape)], dtype=self.dtype)
+                init_prior = np.fromiter((1.0 / self.num_topics for i in xrange(prior_shape)),
+                                         dtype=self.dtype, count=prior_shape)
```

Comment:
This is still vertical indent.

In gensim, we use hanging indent always, consistently: https://www.python.org/dev/peps/pep-0008/#id3